samples/wifi/sr_coex/README.rst

```diff
@@ -61,49 +61,145 @@ The sample uses standard Zephyr threads.
 The threads are configured with non-negative priority (pre-emptible thread).
 For details on threads and scheduling, refer to `Threads`_.
 
-Building and running
-********************
-
-.. |sample path| replace:: :file:`samples/wifi/sr_coex`
+Configuration
+*************
 
-.. include:: /includes/build_and_run_ns.txt
+|config|
 
-The sample can be built for the following configurations:
-
-* Wi-Fi throughput only
-* Bluetooth LE throughput only
-* Concurrent Wi-Fi and Bluetooth LE throughput (with coexistence enabled and disabled mode)
+Setup
+=====
 
 To enable different test modes, set up the following configuration parameters in the :file:`prj.conf` file:
```

Comment:
Should this also be a part of "Configuration files" instead of "Setup"?
```suggestion
To enable different test modes, set up the following configuration parameters in the :file:`prj.conf` file:
```

Reply:
Now that you mention it, the right place most probably is Configuration files. I thought about it and you confirmed my thoughts. Thanks Richa!